README.md

```diff
@@ -0,0 +1,195 @@
+## Creating the DUH document
+NOTE: this tutorial was made using version 1.15.0 of DUH
+
+The DUH document is a JSON5 file that describes an IP block similar to IPXACT.
+The DUH JSON schema is defined [here](https://github.com/sifive/duh-schema).
+DUH is also a suite of tools that help users author DUH documents and generate
+useful artifacts from them. This section describes how to use some of the DUH
+tools to author a DUH document for the PIO block.
+
```

Comment:
We have a document with similar content. https://github.com/sifive/duh/blob/master/README.md
Why we are writing a new one?

Reply:
this README describes the steps to onboard the PIO block. it is not intended to be comprehensive documentation of DUH. I've added a link to the duh REAMDE